packages/components/data-table-manager/package.json

```diff
@@ -40,7 +41,8 @@
     "lodash": "4.17.20",
     "prop-types": "15.7.2",
     "react-beautiful-dnd": "13.0.0",
-    "react-required-if": "1.0.3"
+    "react-required-if": "1.0.3",
```

Comment:
Can we remove the dependency now? Or is still used somewhere else in the component?

Reply:
It's still used, I'll see if I can remove it when we refactor the tests, etc.